components/list/list-item.js

```diff
@@ -31,327 +8,20 @@ const defaultBreakpoints = [842, 636, 580, 0];
-			}
-			:host([dir="rtl"]) [data-breakpoint="1"] ::slotted([slot="illustration"]) {
-				margin-left: 1rem;
-				margin-right: 0;
-			}
-			[data-breakpoint="2"] ::slotted([slot="illustration"]) {
-				margin-right: 1rem;
-				max-height: 5.1rem;
-				max-width: 9rem;
-			}
-			:host([dir="rtl"]) [data-breakpoint="2"] ::slotted([slot="illustration"]) {
-				margin-left: 1rem;
-				margin-right: 0;
-			}
-			[data-breakpoint="3"] ::slotted([slot="illustration"]) {
-				margin-right: 1rem;
-				max-height: 6rem;
-				max-width: 10.8rem;
-			}
-			:host([dir="rtl"]) [data-breakpoint="3"] ::slotted([slot="illustration"]) {
-				margin-left: 1rem;
-				margin-right: 0;
-			}
-			input[type="checkbox"].d2l-input-checkbox {
-				margin: 1.15rem 0.9rem 1.15rem 0;
-			}
-			d2l-list-item-drag-handle {
-				margin: 0.8rem 0 0.8rem 0.4rem;
-			}
-			:host([dir="rtl"]) input[type="checkbox"].d2l-input-checkbox {
-				margin-left: 0.9rem;
-				margin-right: 0;
-			}
-			:host([selectable]:not([disabled]):not([draggable]):hover) d2l-list-item-generic-layout,
-			:host([selectable]:not([disabled])) d2l-list-item-generic-layout.d2l-focusing,
-			:host([selectable][draggable]:not([disabled])) d2l-list-item-generic-layout.d2l-hovering {
-				background-color: var(--d2l-color-regolith);
-			}
-			:host([selected]:not([disabled])) d2l-list-item-generic-layout {
-				background-color: #F3FBFF;
-			}
-			:host([selected]:not([disabled])) d2l-list-item-generic-layout,
-			:host([selected]:not([disabled])) d2l-list-item-generic-layout.d2l-focusing {
-				border-color: #79B5DF;
-			}
-			:host([selected]:not([disabled])) .d2l-list-item-active-border,
-			:host([selected]:not([disabled])) d2l-list-item-generic-layout.d2l-focusing + .d2l-list-item-active-border {
-				position: absolute;
-				width: 100%;
-				bottom: 0;
-				height: 1px;
-				z-index: 5;
-				transform: rotate(1deg);
-				background: #79B5DF;
-			}
-		`];
-	}
-
-	constructor() {
-		super();
-		this._breakpoint = 0;
-		this.breakpoints = defaultBreakpoints;
-		this._contentId = getUniqueId();
-	}
-
-	get breakpoints() {
-		return this._breakpoints;
-	}
-
-	set breakpoints(value) {
-		const oldValue = this._breakpoints;
-		if (value !== defaultBreakpoints) this._breakpoints = value.sort((a, b) => b - a).slice(0, 4);
-		else this._breakpoints = defaultBreakpoints;
-		this.requestUpdate('breakpoints', oldValue);
-	}
-
-	connectedCallback() {
-		super.connectedCallback();
-		ro.observe(this);
-	}
-
-	disconnectedCallback() {
-		super.disconnectedCallback();
-		ro.unobserve(this);
-	}
-
-	firstUpdated(changedProperties) {
-		this.addEventListener('d2l-dropdown-open', () => this._dropdownOpen = true);
-		this.addEventListener('d2l-dropdown-close', () => this._dropdownOpen = false);
-		this.addEventListener('d2l-tooltip-show', () => this._tooltipShowing = true);
-		this.addEventListener('d2l-tooltip-hide', () => this._tooltipShowing = false);
-		super.firstUpdated(changedProperties);
+		return [super.styles];
```

Comment:
is this actually needed since we aren't adding any styles? The component should just naturally inherit them, right?

Reply:
You are totally right. Although this makes for a good example how someone could have styles.